src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -948,6 +948,7 @@ window.qBittorrent.DynamicTable = (function() {
             this.newColumn("last_activity", "", "QBT_TR(Last Activity)QBT_TR[CONTEXT=TransferListModel]", 100, false);
             this.newColumn("availability", "", "QBT_TR(Availability)QBT_TR[CONTEXT=TransferListModel]", 100, false);
             this.newColumn("reannounce", "", "QBT_TR(Reannounce In)QBT_TR[CONTEXT=TransferListModel]", 100, false);
+            this.newColumn("is_private", "", "QBT_TR(Private)QBT_TR[CONTEXT=TransferListModel]", 100, false);
```

Comment:
Shouldn't it be "private" instead of "is_private"? Or is it not the binding with the data returned by WebAPI?